packages/mcl/src/src/mcl/commands/host_info.d

```diff
@@ -90,9 +95,91 @@ Info getInfo()
     info.hardwareInfo.displayInfo = getDisplayInfo();
     info.hardwareInfo.graphicsProcessorInfo = getGraphicsProcessorInfo();
 
+    if (params.codaApiToken) {
+        auto docId = "0rz18jyJ1M";
+        auto hostTableId = "grid-b3MAjem325";
+        auto cpuTableId = "grid-mCI3x3nEIE";
+        auto memoryTableId = "grid-o7o2PeB4rz";
+        auto motherboardTableId = "grid-270PlzmA8K";
+        auto gpuTableId = "grid-ho6EPztvni";
+        auto storageTableId = "grid-JvXFbttMNz";
+        auto osTableId = "grid-ora7n98-ls";
```

Comment:
Are these specifics for the table you edit? Isn't it better to obtain them from some env var?

Reply:
These are specific tables in coda, in a specific document. So the ids need to be hardcoded.

Reply:
@monyarm I agree about these being doc specific, but they still shouldn't be hard coded. * the `docId` should be an env / cli parameter.
* ideally your code will list the tables in the document and locate the table ids by their name
  * if not easily doable, these should also be configurable via env or cli

Reply:
I dunno, given that we want people's hardware info to show up in coda after they run host_info, that seems like a bad idea. I mean, do you expect everyone to look up the coda document id, and add it as an env variable?
I can see wether using names instead of ids is supported for tables, it is for columns, but I'm not sure about the rest.